ci/scripts/opp-env.sh

```diff
@@ -71,6 +71,35 @@ echo "OPP_RENAMED_FILES=$OPP_RENAMED_FILES"
 echo "OPP_REMOVED_FILES=$OPP_REMOVED_FILES"
 echo "OPP_LABELS=$OPP_LABELS"
 
+# The values are passed as env vars.
+# According to https://linux.die.net/man/2/execve,
+# env vars take up Bash's buffers which can lead
+# to error "Argument list too long".
+# Therefore we have to undo the exporting by
+# unsetting them before running any non-builtins
+tmp="$OPP_ADDED_MODIFIED_FILES"
+unset OPP_ADDED_MODIFIED_FILES
+OPP_ADDED_MODIFIED_FILES="$tmp"
```

Comment:
Any reason why `export -n OPP_ADDED_MODIFIED_FILES` would not work? Same for the other variables.

Reply:
I simply did not know this was an option :sweat_smile: I will experiment with this and try to improve it like this :+1: